README.md

```diff
@@ -213,17 +213,19 @@ the types and functions declared herein.
 
 * [matchers](./matchers)
 
+* [paketosbom](./paketosbom): Package paketosbom implements a standardized SBoM format that can be used in Paketo Buildpacks.
+
 * [pexec](./pexec): Package pexec provides a mechanism for invoking a program executable with a varying set of arguments.
 
 * [postal](./postal): Package postal provides a service for resolving and installing dependencies for a buildpack.
 
-* [scribe](./scribe)
+* [sbom](./sbom)
```

Comment:
Should we have the same blurbs you added for `sbom` and `servicebindings` packages in these locations?